mw_basic_auth.go

```diff
@@ -79,7 +79,9 @@ func (k *BasicAuthKeyIsValid) basicAuthHeaderCredentials(w http.ResponseWriter,
 	logger := k.Logger().WithField("key", obfuscateKey(token))
 	if token == "" {
 		// No header value, fail
-		logger.Info("Attempted access with malformed header, no auth header found.")
+		if !k.Spec.BasicAuth.ExtractFromBody {
+			logger.Info("Attempted access with malformed header, no auth header found.")
```

Comment:
I think this message should be ERROR or WARN level (not info).
also, it seems to be extra have it here with reference to out auth extraction from body.
how about moving this log message to `ProcessRequest` and put this warning in log in appropriate situation ( `if k.Spec.BasicAuth.ExtractFromBody` is already there).

Reply:
@dencoded Updated, can you review?

Reply:
Think we don't need to check for `token` with the new change, this scenario will occur when `basicAuthHeaderCredentials` fails.